bytecomp/printlambda.ml

```diff
@@ -73,6 +77,8 @@ let boxed_integer_mark name = function
   | Pnativeint -> Printf.sprintf "Nativeint.%s" name
   | Pint32 -> Printf.sprintf "Int32.%s" name
   | Pint64 -> Printf.sprintf "Int64.%s" name
+  | Puint32 -> Printf.sprintf "Uint32.%s" name
+  | Puint64 -> Printf.sprintf "Uint64.%s" name
```

Comment:
(I wondered where all names that would be passed as the `name` argument existed in the Uint{32,64} modules. This is not the case (for example the `Pnegbint` case would result in `Uint32.neg` being printed), but this is not actually an invariant that the current printer code respects (the `bswap` function does not exist in any of those modules), so I don't think it is worth making the code more complex to prevent that.)